OpenSim/Simulation/Model/Frame.h

```diff
@@ -98,8 +98,16 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Frame, ModelComponent);
         SimTK::Stage::Position);
     OpenSim_DECLARE_OUTPUT(velocity, SimTK::SpatialVec, getVelocityInGround,
         SimTK::Stage::Velocity);
+    OpenSim_DECLARE_OUTPUT(angularVelocity, SimTK::Vec3, getAngularVelocityInGround,
+        SimTK::Stage::Velocity);
+    OpenSim_DECLARE_OUTPUT(linearVelocity, SimTK::Vec3, getLinearVelocityInGround,
+        SimTK::Stage::Velocity);
     OpenSim_DECLARE_OUTPUT(acceleration, SimTK::SpatialVec, getAccelerationInGround,
         SimTK::Stage::Acceleration);
+    OpenSim_DECLARE_OUTPUT(angularAcceleration, SimTK::Vec3, getAngularAccelerationInGround,
+        SimTK::Stage::Acceleration);
+    OpenSim_DECLARE_OUTPUT(linearAcceleration, SimTK::Vec3, getLinearAccelerationInGround,
+        SimTK::Stage::Acceleration);
```

Comment:
We decided to use a lower-case-with-underscores syntax for output names, to match the syntax for property names.